### ISSUE TITLE
Overlay chat in OBS scene

### ISSUE THREAD

**Issue by spprod35** [Durss/Twitchat#20]
Hello

Unless I am mistaken, I cannot find the chat overlay that can be placed in a browser source to display the chat to our viewers.

Does this feature still exist?

Thx for your work

**Comment by Durss** [comment]
Hello !

It was possible at the very beggining but it was poorly made and required lots of other things to be made before to make it cleaner. I removed it long time ago.
Now all those necessary things are in place so that could be done.
I actually completely forgot that in the meantime but I'll most probably add that sooner or later.

Were you using it when it was available or did you see a reference of it somewhere ?

Thank you for your feedback !

**Comment by spprod35** [comment]
Thank you for your feedback, I used it when it was in place, I stopped using it because we wanted to integrate it into our daily show.

However, I've just gone through the new features you released, it's really great.

PS: I'm susu on twitch ;)

**Comment by Durss** [comment]
I'll try to get it back soon :)

_(PS: thank you for your donation that's more than appreciated 🥰❤️ )_

**Comment by theredfish** [comment]
Hi there! 

Thanks for the soft; I use it a lot for Indiedevcasts ! 

A chat overlay is something I would need too, especially if it can mix YT + Twitch. I stream on both platforms at the same time and my viewers can't see each other chat. I would be pleased to have such feature!

**Comment by theredfish** [comment]
@SecareLupus Yep that's true - but at the same time accounts with 2/3k viewers do it. So... 😆 .

 It could be implemented in a way that we can select the chat sources.
